cli/cli.go

```diff
@@ -382,25 +398,34 @@ func Priam(args []string, defaultCfgFile string, infoW, errorW io.Writer) {
 		},
 		{
 			Name: "login", Usage: "gets an access token as a user or service client",
-			ArgsUsage:   "<name> [password]",
-			Description: "if password is not given as an argument, user will be prompted to enter it",
+			ArgsUsage:   "[name] [password]",
+			Description: "if name or password are not given as arguments, user will be prompted",
```

Comment:
Wording is confusing to me: if no arguments are given or if only name is given, user will be prompted